semgrep-core/tests/dockerfile/from-alias-reuse.dockerfile

```diff
@@ -0,0 +1,7 @@
+# MATCH:
+FROM a as b
+FROM b
+
+# MATCH:
```

Comment:
would be nice to also have negative test cases.

Reply:
done